### PR TITLE
Categories with squashed changes

### PR DESCRIPTION
**Pull request by scompo**
Created a fresh PR from #159.

### CODE REVIEW

**Comment by SQiShER** [comment]
This looks great. I was just about to merge, but the commit message doesn't explain what happened in this commit. Could you rephrase it so that it doesn't say "squashed changes" and instead describes the actual changes you made? I could probably rewrite it myself, but I'm not sure how to do that without disassociating the changes from your Github user. And that would be a shame.

The commit message doesn't need to be fancy. I usually stick to this structure:

```
Short summary (max. 72 chars wide) followed by a blank line

Long description if necessary (max. 72 chars per line, so it looks good in the git log)
Long description can have multiple lines.
```

You can change your message by running `git commit --amend` and when you're done just force-push it with `git push --force`.

**Comment by scompo** [comment]
Hope this is fine, and thanks for the pointers on editing commits on git, I usually I just don't edit history, but it's a nice feature, I'll look into that in more detail :+1: for the future

**Comment by SQiShER** [comment]
This is great! Thanks for this!

**Comment by scompo** [comment]
No problem, glad I could help :)
Just a question, do you think I should delete the branch from the old PR or should I leave it there for documentation reference?

**Comment by SQiShER** [comment]
Once it has been merged, it isn't needed anymore. Feel free to delete it. :smiley:

**Comment by scompo** [comment]
Done, thanks :+1:
